calblueprint/vouchers4veggies
# [scanning] Update scanner design and implement form to add vouchers manually (screens/scanning/ManualVoucherScreen)

# ✨ New in this PR ✨

## One liner: what did you do? 🚨 
- Updated designs for the barcode scanner
- Implemented toast notifications for successful scans
- Created a new ManualVoucherScreen for adding vouchers through a form (not functional yet)

## Coverage 🙆‍♀️
- [x] Scan button flow.
- [x] Confirmation message on successful scans.
- [x] Manually add transactions.
- [x] Remove redundant ScannerScreen (screens/scanner/ScannerScreen.tsx)

## Future tasks
- Store voucher data in a state variable + link to Firebase.
- Implement dropdown in the Add Manually form.
- Link to Noah's Review & Submit screen.

## How can the reviewer test your code? 👩‍💻
Run `npm start` and navigate to the scanning screen. Check if barcodes scan properly + a notification is displayed only on successful scans. Also, check if the Add Manually button navigates properly.

## Any bugs you encountered or still having trouble with? 🐛
This is a huge PR: should've been broken down into 2 or more (my b).

## Resources 📔
External component for displaying toasts: https://github.com/calintamas/react-native-toast-message

🥦 CC: @allisonhongberkeley @EthanAuyeung
